llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -1621,6 +1626,61 @@ class UnaryInstruction
   }
 };
 
+class ExtractValueInst : public UnaryInstruction {
+  /// Use Context::createExtractValueInst() instead.
+  ExtractValueInst(llvm::ExtractValueInst *EVI, Context &Ctx)
+      : UnaryInstruction(ClassID::ExtractValue, Opcode::ExtractValue, EVI,
+                         Ctx) {}
+  friend Context; // for ExtractValueInst()
+
+public:
+  static Value *create(Value *Agg, ArrayRef<unsigned> Idxs, BBIterator WhereIt,
+                       BasicBlock *WhereBB, Context &Ctx,
+                       const Twine &Name = "");
+
+  static bool classof(const Value *From) {
+    return From->getSubclassID() == ClassID::ExtractValue;
+  }
+
+  static Type *getIndexedType(Type *Agg, ArrayRef<unsigned> Idxs) {
```

Comment:
nit: It is not straight forward what this function does, so I would copy LLVM IR's comments.

Reply:
Done.